tests/console/yast2_dns_server.pm

```diff
@@ -25,10 +25,10 @@ sub assert_running() {
     my $running = shift;
 
     if ($running) {
-        assert_script_run 'systemctl is-active named | grep -E "^active"';
+        assert_script_run '(systemctl is-active named || true) | grep -E "^active"';
```

Comment:
Yes, in this case this might be enough but I kept it to be more comparable to the next check. Both ways can work. I have a bigger refactoring prepared anyway so even if I don't change it now it will be changed later.